gateway-protocol/src/main/proto/gateway.proto

```diff
@@ -155,7 +155,37 @@ message CreateProcessInstanceWithResultResponse {
   // JSON document
   // consists of visible variables in the root scope
   string variables = 5;
+}
 
+message EvaluateDecisionRequest {
+  // the unique key identifying the decision (e.g. returned from a decision
+  // in the DeployResourceResponse message)
+  int64 decisionKey = 1;
+  // the decision ID
+  string decisionId = 2;
+  // JSON document that will instantiate the variables for the decision; it
+  // must be a JSON object, as variables will be mapped in a key-value fashion.
+  // e.g. { "a": 1, "b": 2 } will create two variables, named "a" and
+  // "b" respectively, with their associated values. [{ "a": 1, "b": 2 }] would
+  // not be a valid argument, as the root of the JSON document is an array and
+  // not an object.
+  string variables = 3;
+}
+
+message EvaluateDecisionResponse {
+  // the unique key identifying the decision (e.g. returned from a decision
+  // in the DeployResourceResponse message)
+  int64 decisionKey = 1;
+  // the decision ID
+  string decisionId = 2;
+  // the version of the decision which was evaluated
+  int32 version = 3;
```

Comment:
🔧 Let's make these comments a bit more consistent by adding `the decision which was evaluated` to each of these, like `version`

Reply:
:question: What do you think about adding `the decision to be evaluated` in all the comments of the `EvaluateDecisionRequest` as well?

Reply:
That would help clarify things. Sounds good to me 👍